content/en/docs/Responses/AlbumID3.md

```diff
@@ -127,6 +128,7 @@ description: >
 | `originalReleaseDate` | [`ItemDate`](../itemdate) | No |   **Yes**   | Date the album was originally released. |
 | `releaseDate` | [`ItemDate`](../itemdate)  | No |   **Yes**   | Date the specific edition of the album was released. *Note:* for files using ID3 tags, releaseDate should generally be read from the **TDRL** tag. Servers that use a different source for this field should document the behavior. |
 | `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation. |
+| `explicitStatus` | `string` | No |  **Yes**    | Returns "explicit" if at least one song is explicit, "clean" if no song is explicit and at least one is "clean" else "". |
```

Comment:
Needs to list the new field in the OpenSubsonic New Fields section below.

Reply:
Too long since my last commit :)